test/main/test.js

```diff
@@ -119,3 +119,17 @@ QUnit.skip( "test blocks are skipped", function( assert ) {
 } );
 
 QUnit.skip( "no function" );
+
+QUnit.module( "Missing Callbacks" );
+
+QUnit.test( "QUnit.test without a callback logs a descriptive error", function( assert ) {
+	assert.throws( () => {
```

Comment:
Should be a normal function as `PhantomJS` doesn't support arrow functions. The same goes for the other test.